lint/build.gradle

```diff
@@ -1,14 +1,12 @@
-apply plugin: 'java-library'
-
-configurations {
-    lintChecks
-}
+apply plugin: 'kotlin'
 
 dependencies {
     compileOnly 'com.android.tools.lint:lint-api:26.0.1'
-    compileOnly 'com.android.tools.lint:lint-checks:26.0.1'
-}
+    compileOnly 'org.jetbrains.kotlin:kotlin-stdlib-jre7:1.2.10'
```

Comment:
We had some trouble before when we started to use Java 8 internally, it made the library not work for developers on Java 7.

Are we 100% sure this won't bubble up in any way>

Reply:
Yeah, I believe that was with EasyPermissions, right? Anyway, I have a hard time seeing how this could affect android app modules since this is a pure Java (or Kotlin) lib. I remember the issue occurring when we set the target compatibility in the `android` block whereas this is going to be run by IntelliJ. Also, if our `app` module doesn't compile, we'll know we messed up.

Reply:
Do we really need `jre8`?